templates/zerver/help/mute-a-user.md

```diff
@@ -0,0 +1,75 @@
+# Mute a user
+
+!!! tip ""
+    This feature works on an individual basis. In case of a user seriously
+    violating the code of conduct of your community, consider requesting
+    your organization administrator to [deactivate](/help/deactivate-or-reactivate-a-user)
+    them instead.
+
+
+To deal with annoying or troublesome users, you can mute them.
+A user you mute will not know that they have been muted, and muting
+someone will have the following effects:
+
+- Stream messages sent by muted users will be hidden under a
+  **Click to reveal** banner, and [private messages](/help/private-messages)
+  will be completely excluded from the message view.
+
+- Messages sent by a muted user will automatically be
+  [marked as read](/help/marking-messages-as-read) for you, and
+  will never generate any desktop, push or email notifications.
+
+- Muted users will be excluded from the right sidebar, the
+  [private conversations](/help/private-messages) list, and from
+  the [mentions](/help/mention-a-user-or-group) autocomplete.
+
+- Muting a user does not affect their ability to send you messages
+  or view your [status and availability](/help/status-and-availability).
```

Comment:
Can you test what happens when a muted user tries to mention you?  We should document what happens in that case.

Reply:
Nothing special happens for the mutee. Maybe we should document this as a general statement?
Something like `When you mute someone, it will not affect the muted user an any way`?

Reply:
Yeah, I think so.  What I meant is we should document that the message will appear in your Mentions tab, but be already read and not display unless you click.